conf/modules.config

```diff
@@ -18,8 +18,32 @@ process {
         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     ]
 
-    withName: FASTQC {
-        ext.args = '--quiet'
+    withName: BCFTOOLS_CONCAT {
+        ext.args = '-a -O z'
+        ext.prefix = { "${meta.id}_${meta.caller}.concat" }
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: BCFTOOLS_NORM {
+        ext.args = '-m-any -O z'
+        ext.prefix = { "${meta.id}_${meta.caller}.norm" }
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: BCFTOOLS_SORT {
+        ext.prefix = { "${meta.id}_${meta.caller}.norm.sort" }
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: BCFTOOLS_ISEC {
+        ext.args = '-n~11 -w1 -O z'
```

Comment:
should that tilde suppose to be there?

Reply:
yes weird syntax by the tool. Tilde means in exactly those two files